src/utils/potentialDupMsg/index.test.ts

```diff
@@ -40,7 +40,8 @@ describe('potentialDupMsg', () => {
       from: WALLET_ADDRESS,
       value: '0',
       method: 0,
-      nonce: 0
+      nonce: 0,
+      params: ''
```

Comment:
This broke the tests since some of the Message types changed, just fixed it in this PR